src/odemis/gui/cont/views.py

```diff
@@ -330,8 +345,8 @@ def __init__(self, main_data, tab_data, overview_canvas, m_view):
             max_x = ax_x.range[1] - ax_x.range[0]
             max_y = ax_y.range[1] - ax_y.range[0]
             if max_x < 0.05 and max_y < 0.05:
-                ovv_shape = (2 * int(max_y / self.m_view.mpp.value),
-                             2 * int(max_x / self.m_view.mpp.value), 3)
+                ovv_shape = (int(max_y / self.m_view.mpp.value),
+                             int(max_x / self.m_view.mpp.value), 3)
             else:
                 ovv_shape = (int(OVV_DIMENSION[1] / self.m_view.mpp.value),
                              int(OVV_DIMENSION[0] / self.m_view.mpp.value), 3)
```

Comment:
For now ovv_shape can be of arbitrary size. It depends on the view mpp and axis range. However, in practice, you don't want something > 1200 px, otherwise it takes too much memory, and anything below < 1000 is a waste and could have been zoomed. So I'd recommend to pick a constant size (~1200 px), and adapt the image and view mpp so that it fits the screen.